generators/app/templates/_package.json

```diff
@@ -25,6 +28,11 @@
     "cordova:build": "ngx-scripts cordova build --device --release --copy dist",
     "cordova:clean": "ngx-scripts clean",
     "cordova": "cordova",
+<% } -%>
+<% if (props.target.includes('electron')) { -%>
+    "electron:run": "electron .",
+    "electron:build": "npm run env -s && ng build --prod --base-href ./",
+    "electron:package:win" : "electron-packager . --overwrite --platform=win32 --arch=ia32 --out=out --icon=src/assets/icon.ico",
```

Comment:
use a dist-electron folder? (+ add it to .gitignore)

Then it would be nice to add win64/linux/mac commands, along with a `electron:package` to run them all at once.
Also, with this you should add a question like with cordova to choose target platforms ie: win32, win64, linux, osx then the PR would be complete :)